lte/gateway/deploy/roles/dev_common/tasks/main.yml

```diff
@@ -100,6 +100,18 @@
     line: "alias disable-tcp-checksumming='sudo ethtool --offload eth1 rx off tx off; sudo ethtool --offload eth2 rx off tx off'"
   when: full_provision
 
+- name: Create convenient aliases for magma services
```

Comment:
are these aliases also created on magma_test?

Reply:
Actually yes, the `dev_common` role is also used for the magma_test VM. Would it be better to create a new role for this?

Reply:
You could also work with tags. But imo we are already using them too much - and it becomes confusing. A new role is a good idea.

Reply:
Created a new role